doc/cephfs/shell.rst

```diff
@@ -0,0 +1,277 @@
+
+ Options:
+  -l	   list with long format - show permissions
+  -r        reverse sort     
+
+
+cat
+---
+
+ Concatenate files and print on the standard output
+
+ Usage : cat  <file>....
+        Where
+             file : name of the file
+
+cd
+--
+
+ Open a specific named directory.
+
+ Usage : cd <directory>
+        Where
+            directory - path/directory name 
+
+cwd
+---
+ Get current working directory.
+ 
+ Usage : cwd
+
+
+quit/Ctrl + D
+-------------
+ Close the shell.
+
+chmod
+-----
+ Change the permissions of file/directory.
+ 
+ Usage : chmod <file/directory> <mode>
+
+mv
+--
+
+ Moves files/Directory from source to destination.
+
+ Usage : mv <source_path> <destination_path>
+
+rmdir
+-----
+
+ Delete a directory(ies).
+
+ Usage : rmdir <directory_name>.....
+
+rm
+--
+
+ Remove a file(es).
+
+ Usage : rm <file_name>...
+
+
+write
+-----
+
+ Create and Write a file.
+
+ Usage : write <file_name>
+          <Enter Data>
+          Ctrl+D Exit.
+
+lls
+---
+
+ Lists all files and directories in the specified directory.Current local directory files and directories are listed if no     path is mentioned
+
+ Usage: lls <path>.....
+
+lcd
+---
+
+ Moves into the given local directory.
+
+ Usage : lcd <path>
+
+lpwd
+----
+
+ Prints the absolute path of the current local directory.
+
+ Usage : lpwd
+
+
+umask
+-----
+ set and get the file mode creation mask 
+
+ Usage : umask <mode>
+
+alias
+-----
+
+ Define or display aliases
+
+ Usage: alias [name] | [<name> <value>]
+     Where:
+         name - name of the alias being looked up, added, or replaced
+         value - what the alias will be resolved to (if adding or replacing)
+                 this can contain spaces and does not need to be quoted
+
+pyscript
+--------
+
+ Runs a python script file inside the console
+
+ Usage: pyscript <script_path> [script_arguments]
+
+ Console commands can be executed inside this script with cmd("your command")
+ However, you cannot run nested "py" or "pyscript" commands from withinthis script
+ Paths or arguments that contain spaces must be enclosed in quotes
+
+py
+--
+ Invoke python command, shell, or script
+
+ Usage : py <command>: Executes a Python command.
+         py: Enters interactive Python mode.
+
+shortcuts
+---------
+
+ Lists shortcuts (aliases) available
+
+history
+-------
+
+ View, run, edit, and save previously entered commands
+
+ Usage : history [-h] [-r | -e | -s | -o FILE | -t TRANSCRIPT] [arg]
+
+ Options:
+   -h            show this help message and exit
+   -r            run selected history items
+   -e            edit and then run selected history items
+   -s            script format; no separation lines
+   -o FILE       output commands to a script file
+   -t TRANSCRIPT output commands and results to a transcript file
+
+unalias
+-------
+
+ Unsets aliases
+ 
+ Usage : unalias [-a] name [name ...]
+     Where:
+         name - name of the alias being unset
```

Comment:
See: http://docs.ceph.com/docs/master/man/8/ceph/. This style eliminates the repeated `Where:`

For example: 
```
Subcommand remove removes monitor named <name>.
Usage:
   ceph mon remove <name>
```